connman/src/rtnl.c

```diff
@@ -1126,35 +1154,66 @@ static bool is_route_rtmsg(struct rtmsg *msg)
 		return false;
 
 	if (msg->rtm_protocol != RTPROT_BOOT &&
-			msg->rtm_protocol != RTPROT_KERNEL)
+					msg->rtm_protocol != RTPROT_KERNEL) {
+
+		if (handle_rtprot_ra && msg->rtm_protocol == RTPROT_RA)
```

Comment:
Please double-check use of `rtnl->handle_rtprot_ra` vs global `handle_rtprot_ra`. It wasn't obvious to me.

Reply:
The changes 324012e324bcea0cab51a5758a84e9d242a5ec94 + a46ed14c2d6c6a596f03c62602ea0bb5d5d1ad20 + e07773823fa3b1527d12f00781339e61165028a9 make this go the other way. And as I suggested in https://github.com/sailfishos/connman/pull/29#discussion_r909543189 it might be better + more generic way to add a functionality to support the varying kernel netlink user type protocols to be handled.